src/CoreWCF.Primitives/src/CoreWCF/Description/OperationDescription.cs

```diff
@@ -124,9 +124,9 @@ public string Name
             get { return Behaviors; }
         }
 
-        internal KeyedByTypeCollection<IOperationBehavior> Behaviors { get; }
+        public KeyedByTypeCollection<IOperationBehavior> Behaviors { get; }
 
-        internal XmlName XmlName { get; }
+        public XmlName XmlName { get; }
 
```

Comment:
You don't need to expose this. You can do the same thing as I mentioned for MessagePartDescription using XmlConvert.DecodeName on the Name property.

Reply:
This is addressed in 75f6cfa72e482b0f0f42ad137be5f9b694044554.